java/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -361,3 +361,12 @@ mau.error.description=we can't log you in right now because too many users are s
 cookie.policy=Cookie Policy
 cookie.consent.banner.content=We use cookies to ensure that you get the best overall experience. These cookies are used to maintain an uninterrupted continuous session whilst providing smooth and personalized services. To learn more about how we use cookies, refer our
 cookie.consent.banner.clear=Got it
+
+# Duo Authenticator
+
+error.failed.with.duo=Failed Authentication with Duo
+error.user.not.registered=Couldn't get the user information. User may not be registered in Duo.
+error.user.not.found=Couldn't get the verified user name.
+error.mobile.not.found=User may not have a mobile number in user's profile for Duo Authentication. Please update it.
+error.mobile.not.found.duo=Unable to get the registered phone number from Duo. Authentication failed.
+error.number.mismatch.duo=Authentication failed due to mismatch in mobile numbers. Please update the correct registered duo mobile number in user profile.
```

Comment:
Since these error messages are visible to the end user, shall we get these reviewed?